website/content/tutorials/1-getting-started/_index.md

```diff
@@ -1,52 +1,53 @@
+Once we understand the required information structure for a computer, how do we specify a model in Metaschema and what are the benefits of using a Metaschema-based approach to move towards a set of equivalent data models?
 
-{{<mermaid>}}
+## Metaschema Concepts
+
+Metaschema helps developers to define information models once as a [Metaschema module](../../specification/glossary/#metaschema-module). A Metaschema module specifies the information elements of the information model and how that model is represented in supported machine-readable data formats (i.e. JSON, YAML, XML), which we call a [data model](/specification/glossary/#data-model). A document in such a format is an instance of that module. A schema can be used to check that a document is well-formed and valid against the model represented by the format. Such schemas can be derived deterministically and programmatically from a Metaschema module (or "metaschema").
 
+```mermaid
 erDiagram
 
-  "Metaschema definition" }|..|{ "instance" : "must specify model of"
-  "Metaschema tool" }|..|{ "Metaschema definition" : "must parse"
+  "Metaschema module" }|..|{ "instance" : "must specify model of"
+  "Metaschema tool" }|..|{ "Metaschema module" : "must parse"
   "Metaschema tool" }|..|{ "instance" : "can parse"
   "Metaschema tool" }|..|{ "schema" : "can generate"
   "schema" }|..|{ "instance" : "must validate"
+```
 
-{{</mermaid>}}
-
-## Basic Modeling and Basic Metaschema Defintions
+## Basic Modeling and Basic Metaschema Module
 
-We start with an empty Metaschema definition, like the one below, saved in a file called `computer_metaschema.xml`.
+We start with an empty Metaschema module, like the one below, saved in a file called `computer_metaschema.xml`.
 
 ```xml
 <?xml version="1.0" encoding="UTF-8"?>
 <METASCHEMA xmlns="http://csrc.nist.gov/ns/oscal/metaschema/1.0">
 </METASCHEMA>
 ```
 
-Metschema definitions, like the one above, are in XML. A definition begins and ends with capitalized `METASCHEMA` tags. This definition is an empty file, and it is not a valid, well-formed defintion. It is simply the base we will start with. Within those beginning and ending tags, we want to add useful metadata for both developers and Metaschema-enabled tools to consume this definition, as below.
+Metschema modules, like the one above, are expressed in XML. A module begins and ends with capitalized `METASCHEMA` tags. This module is an empty file, which means it is not a valid, well-formed module, since it lacks any of the required data elements. It is simply the base we will start with. Within the beginning and ending tags, we want to add the required and useful metadata needed for both developers and Metaschema-enabled tools to consume this module. We do this below.
```

Comment:
Minor, non-blocking: I think we meant Metaschema.

```suggestion
Metaschema modules, like the one above, are expressed in XML. A module begins and ends with capitalized `METASCHEMA` tags. This module is an empty file, which means it is not a valid, well-formed module, since it lacks any of the required data elements. It is simply the base we will start with. Within the beginning and ending tags, we want to add the required and useful metadata needed for both developers and Metaschema-enabled tools to consume this module. We do this below.
```